packages/api/cms-api/src/dam/files/files.resolver.ts

```diff
@@ -116,6 +124,8 @@ export function createFilesResolver({ File, Scope: PassedScope }: { File: Type<F
         }
 
         @Mutation(() => [File])
+        @AffectedEntity(Folder, { idArg: "targetFolderId", nullable: true })
```

Comment:
`nullable` is needed for this case because `targetFolderId` can be null (when moving to the root folder). The scope is then the current scope of each file. So checking for file scope is sufficient.